src/client/javascripts/App.js

```diff
@@ -35,19 +40,28 @@ class App extends Component {
   render() {
     return (
 
-           // Render the screen on View.
+          //  Render the screen on View.
            <View style={styles.container}>
-                                  <Text style={styles.welcome}>
-                      Welcome to React Native, {this.state.os} !
-                                  </Text>
-                                  <Text style={styles.instructions}>
-                      To get started, edit index.ios.js
-                                  </Text>
-                                  <Text style={styles.instructions}>
-                      Press Cmd+R to reload,                    {'\n'}
-                      Cmd+D or shake for dev menu
-                                  </Text>
-                          </View>
+
+              {/* Render facebook login button */}
+             <LoginButton
+                onLoginFinished={
+                  (error, result) => {
+                    if (error) {
+                      alert('login has error: ' + result.error);
```

Comment:
Where does this alert show? To the user?

In any case, capitalize Login and all user-visible / debug strings everywhere.